src/Resizer.cs

```diff
@@ -1,27 +1,56 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.IO;
-using ImageSharp;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats;
+using SixLabors.ImageSharp.Formats.Bmp;
+using SixLabors.ImageSharp.Formats.Gif;
+using SixLabors.ImageSharp.Formats.Jpeg;
+using SixLabors.ImageSharp.Formats.Png;
+using SixLabors.Primitives;
 
 namespace ImageResizer
 {
-    public static class Resizer
+    public class Resizer
     {
-        public static void Resize(string filePath, IEnumerable<int> widths)
+        private Image<Rgba32> _image;
```

Comment:
mutable fields make me feel icky, can we make these readonly and have them populated from a constructor that takes the `filePath` as an arg